packages/vite-plugin-angular/src/lib/angular-vite-plugin.ts

```diff
@@ -122,7 +122,7 @@ export function angular(options?: PluginOptions): Plugin[] {
         await buildAndAnalyze();
       },
       async handleHotUpdate(ctx) {
-        if (/\.[cm]?tsx?$/.test(ctx.file)) {
+        if (/\.[cm]?ts?$/.test(ctx.file)) {
```

Comment:
Remove the question mark also

```suggestion
        if (/\.[cm]?ts$/.test(ctx.file)) {
```